trackintel/preprocessing/triplegs.py

```diff
@@ -221,7 +234,7 @@ def _seperate_ids(row):
     # assign trip_id to tpls
     temp = trips.explode("tpls")
     temp.index = temp["tpls"]
-    temp = temp[temp["spts"].notna()]
+    temp = temp[temp["spts"].notna()] #todo: what is this dropping exactly? Should be tpls?
```

Comment:
Yes, a mix-up. Although I am suprised that it didn't throw an error in the tests.